Userland/Libraries/LibCore/Process.h

```diff
@@ -8,20 +8,62 @@
 
 #pragma once
 
+#include <AK/DeprecatedString.h>
 #include <AK/Forward.h>
 #include <AK/Span.h>
+#include <LibCore/File.h>
 
 namespace Core {
 
+namespace FileAction {
+
+struct OpenFile {
+    DeprecatedString path;
+    File::OpenMode mode;
+    int fd;
+    int permissions = 0600;
```

Comment:
Do we not have a permission mode enum for files elsewhere in LibCore? if not we should probably have one in the future.

Reply:
We don't. I can change the type from `int` to `mode_t`, though.